batchflow/models/torch/base.py

```diff
@@ -559,6 +559,11 @@ def _get_placeholder_shapes(self):
                 if self.classes is None:
                     self.classes = shapes[0][0]
 
+    def _to_device(self):
```

Comment:
Why is it private?

Reply:
There is no obvious reason to make it public in this form.

It is possible to make it public and change its signature to `to_device(self, device=None)`, but in this case we also need to change `self.device` and process all the model-related object, e.g. move loss to another device if it is an instance of `nn.Module`, reconstruct optimizers. 

Do you see cases when you might want to call this method independently and move a constructed model to another device on the fly?

Reply:
Reflect all these comments in the docstring, please